### PR TITLE
Update Angular 18

### PR DESCRIPTION
**Pull request by whyvrafvr**

### CODE REVIEW

**Comment by whyvrafvr** [comment]
linked to #48 & #49

File: src/app/admin/admin.module.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { CommonModule } from '@angular/common';
 
 import { AdminRoutingModule } from './admin-routing.module';
 import { UsersComponent } from './users/users.component';
-import { NgChartsModule } from 'ng2-charts';
+import { BaseChartDirective, provideCharts, withDefaultRegisterables } from 'ng2-charts';
+import { Legend, Colors } from 'chart.js';
```
Comment (s4m5ep10l) [inline src/app/admin/admin.module.ts:8]: can be removed I guess...

Comment (s4m5ep10l) [reply src/app/admin/admin.module.ts:8]: @whyvrafvr ?

**Review by s4m5ep10l — APPROVED** [review]
lgtm

File: src/app/admin/admin.module.ts (hunk 1 above)
Comment (whyvrafvr) [reply src/app/admin/admin.module.ts:8]: Sure let's merge and I'll remove these imports

Comment (s4m5ep10l) [reply src/app/admin/admin.module.ts:8]: ok docky

**Review by s4m5ep10l — COMMENTED** [review]
lgtm

**Pull request merged**